integration-tests/__tests__/only_changed.test.js

```diff
@@ -73,6 +73,75 @@ test('run only changed files', () => {
   expect(stderr).toMatch(/PASS __tests__(\/|\\)file3.test.js/);
 });
 
+test('report test coverage for only changed files', () => {
+  const windowsSlashRegex = new RegExp(`__tests__\\${path.win32.sep}`, 'g');
```

Comment:
No way to use `slash` instead of this regex?

Reply:
good point

Reply:
It only worked for one of them:

![image](https://user-images.githubusercontent.com/1404810/36350308-c59a3620-1496-11e8-89a6-015d778d98a2.png)

Reply:
Looking at slash source, it's because the text contains non-ascii chars https://github.com/sindresorhus/slash/blob/master/index.js#L10

Reply:
The test above uses `expect(stderr).toMatch(/PASS __tests__(\/|\\)file2.test.js/);` insead of snapshot, might be a better idea

Reply:
Agreed, it's enough to see what's going on.

Reply:
I agree the snapshots is not necessary here.

IMO all we need to know is these two assertions:
```js
({stdout} = runJest(DIR));
expect(stdout).toMatch('b.js');
({stdout} = runJest(DIR, ['-o']));
expect(stdout).not.toMatch('b.js');
```

Do you agree? That should work just as well on windows as *nix 😄

Reply:
I'm down with that

Reply:
there 🙂